dev_guide/master_node_configuration.adoc

```diff
@@ -17,16 +17,16 @@ The config files are fully specifying with no defaulting.  This means that any e
 == Create the starting config files
 The `openshift start` command accepts flags that indicate that it should simply write the config file that it would have used and terminate.  This is useful for getting a starting point for the config.  You can do this by running 
 
-* `openshift start --writeconfig --master-config=master.yaml --node-config=node.yaml`
-* `openshift start master --write-config --config=master.yaml`
-* `openshift start node --write-config --config=node.yaml`
+* `openshift start --write-config=openshift.local.config
+* `openshift start master --write-config=openshift.local.config/master`
+* `osadm create-node-config --node-dir=openshift.local.config/node-myhost --node=myhost --hostnames=myhost.local`
```

Comment:
maybe add an example IP to the --hostnames arg, just so people know they can (and probably should) include the node IP